specs/interop/predeploys.md

```diff
@@ -416,6 +426,30 @@ dependency set called `dependencySet()`. This function MUST return the array of
 `L1Block` MUST also provide a public getter to get the dependency set size called `dependencySetSize()`. This function
 MUST return the length of the dependency set array.
 
+### Deposit Context
+
+The `L1Block` contract is updated with the following methods to support [deposit contexts](./derivation.md#deposit-context).
+
+#### `isDeposit`
+
+```solidity
+function isDeposit() (return bool)
+```
+
+A new method added to `L1Block` indicating whether the current execution is in a [deposit context](./derivation.md#deposit-context). The method has the following signature:
+
+
+Only the `CrossL2Inbox` is authorized to call `isDeposit`.
```

Comment:
Could we add a follow up sentence with the rationale? ie if any contract could call this, then apps can build censorship easily